modules/end-user-guide/partials/proc_creating-a-workspace-by-importing-the-source-code-of-a-project.adoc

```diff
@@ -107,35 +148,54 @@ image::workspaces/configure-command.png[Configure tasks]
 
 
 [id="importing-to-a-running-workspace-using-the-git-clone-command_{context}"]
-== Importing to a running workspace using the *Git: Clone* command
+== Importing a project with *Git: Clone* in the GUI
+
+You can import your project into a running workspace with the *Git: Clone* command in the graphic user interface of {prod-short}.
+ 
+.Prerequisites
+* A started workspace
 
-To import to a running workspace using the *Git: Clone* command:
+.Procedure
 
-. Start a workspace, then use the *Git: Clone* command from the command palette or the Welcome screen to import a project to a running workspace.
+Import your project's codebase into a running workspace as follows:
+
+. Select the *Git: Clone* command on the *Welcome* screen or from the command palette:
+* On the *Welcome* screen, click the *Git: Clone* link.
 +
 image::workspaces/{project-context}-welcome.png[Welcome screen]
 
-. Open the command palette by using `F1`, `CTRL-SHIFT-P`, or the link displayed on the Welcome screen.
+* To open the command palette, press F1 or Ctrl+Shift+P.
 +
 image::workspaces/git-clone-command.png[Invoke git clone command]
 
-. Enter the path to the project that is about to be cloned.
+. Enter the path to the project to be cloned.
 +
 image::workspaces/git-clone-command-2.png[Configure git clone command]
 
 
 [id="importing-to-a-running-workspace-with-git-clone-in-a-terminal_{context}"]
-== Importing to a running workspace with `git clone` in a terminal
+== Importing a project with `git clone` in a terminal
+
+You can use the command line to import your project into a running workspace.
 
-In addition to the approaches above, you can also start a workspace, open a *Terminal*, and type `git clone` to pull code.
+.Prerequisites
+* A running workspace
+
+.Procedure
+
+Import your project's codebase into the running workspace as follows:
 
+. Open a terminal inside the running workspace.
+
+. Type the `git clone` command to pull code.
++
 image::workspaces/git-clone-terminal.png[Run git clone in a terminal]
 
 [NOTE]
 ====
 Importing or deleting workspace projects in the terminal does not update the workspace configuration, and the IDE does not reflect the changes in the *Devfile* tab in the dashboard.
 
-Similarly, when you add a project using the *Dashboard*, then delete it with `rm -fr myproject`, it may still appear in the *Devfile* tab.
+Similarly, when you add a project using the *Dashboard* and then delete it with `rm -fr myproject`, it may still appear in the *Devfile* tab.
```

Comment:
"the project may still appear" would be very clear.

Reply:
Yep, rewrote that sentence.